test/integration/start_stop_delete_test.go

```diff
@@ -38,6 +38,7 @@ func TestStartStop(t *testing.T) {
 		{"nocache_oldest", []string{
 			"--cache-images=false",
 			fmt.Sprintf("--kubernetes-version=%s", constants.OldestKubernetesVersion),
+			"--kvm-network=default",
```

Comment:
is the value "default" special in any way? If not, I'd prefer it to be something that is unlikely to conflict or appear special, like "test-net".

Reply:
Nope, it's not. I'll change it to `test-net`. :)

Reply:
Actually, I take this back. As the integration test does start minikube,  the `--kvm-network=net-test` won't exist, it will fail. Unless we create this network on the CI beforehand, but probably we should have integration tests for both cases. Using the default network, using the pre-defined network.